src/AzureExtension/Assets/QuickStartPlayground/docsEmbeddings-OpenAI.json

```diff
@@ -1549,7 +1549,7 @@
         "name": "cpp-helloworld",
         "codespaces": "\n\n=== ./.devcontainer/devcontainer.json ===\n{\n    \"name\": \"C++\",\n    \"build\": {\n        \"dockerfile\": \"Dockerfile\"\n    },\n    \"customizations\": {\n        // Configure properties specific to VS Code.\n        \"vscode\": {\n            \"settings\": {},\n            \"extensions\": [\n                \"ms-vscode.cpptools\"\n            ]\n        }\n    }\n}\n\n=== ./.devcontainer/Dockerfile ===\nFROM mcr.microsoft.com/devcontainers/base:ubuntu\n\nENV DEBIAN_FRONTEND noninteractive\n\nRUN apt-get update && \\\napt-get install -y g++ gdb gcc clang cmake tar",
         "readme": "\n\n=== ./README ===\n# Set up\n\nRun `g++ -g main.cpp -o main.out`\n\n# Get started\n\nRun `./main.out`",
-        "code": "\n\n=== ./main.cpp ===\n#include <iostream>\n#include <string>\n\nusing namespace std;\n\nint main()\n{\n    cout << \"Hello World\\n\";\n    cout << \"Input: \";\n    string data;\n    getline(cin, data);\n    cout << \"Output: \" << data << \"\\n\\n\";\n    return 0;\n}\n\n=== ./prompt.txt ===\n==== EXAMPLE PROMPT: Create a VS Code Codespaces ready environment to get started with a hello world application using C++ ====\n\n=== ./.vscode/launch.json ===\n{\n    \"version\": \"0.2.0\",\n    \"configurations\": [\n        {\n            \"name\": \"Launch Main\",\n            \"type\": \"cppdbg\",\n            \"request\": \"launch\",\n            \"program\": \"${workspaceFolder}/main.out\",\n            \"args\": [],\n            \"stopAtEntry\": false,\n            \"cwd\": \"${workspaceFolder}\",\n            \"environment\": [],\n            \"externalConsole\": false,\n            \"MIMode\": \"gdb\",\n            \"setupCommands\": [\n                {\n                    \"description\": \"Enable pretty-printing for gdb\",\n                    \"text\": \"-enable-pretty-printing\",\n                    \"ignoreFailures\": true\n                }\n            ],\n            \"preLaunchTask\": \"Build Main\"\n        }\n    ]\n}\n\n=== ./.vscode/tasks.json ===\n{\n    \"version\": \"2.0.0\",\n    \"tasks\": [\n        {\n            \"label\": \"Build Main\",\n            \"type\": \"shell\",\n            \"command\": \"g++ -g main.cpp -o main.out\",\n            \"group\": {\n                \"kind\": \"build\",\n                \"isDefault\": true\n            }\n        }\n    ]\n}",

```

Comment:
Future consideration: the C++ sample appears to be the only one that has a `tasks.json` file included with it. We should assess whether we want to keep (or consider adding it to other sample projects).